ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Pre-defined styles of icons
+@objc(MSFListIconVnextStyle)
+public enum MSFListIconVnextStyle: Int, CaseIterable {
+    case none
+    case iconOnly
+    case large
+}
+
+/// Pre-defined types of icons
+@objc(MSFListAccessoryType)
+public enum MSFListAccessoryType: Int, CaseIterable {
+    case none
+    case disclosure
+    case detailButton
+    case checkmark
+
+    var icon: UIImage? {
+        let icon: UIImage?
+        switch self {
+        case .none:
+            icon = nil
+        case .disclosure:
+            icon = UIImage.staticImageNamed("iOS-chevron-right-20x20")
+        case .detailButton:
+            icon = UIImage.staticImageNamed("more-24x24")
+        case .checkmark:
+            icon = UIImage.staticImageNamed("checkmark-24x24")
+        }
+        return icon
+    }
+}
+
+///Properties that make up cell content
+@objc(MSFListVnextCellData)
+public class MSFListVnextCellData: NSObject, ObservableObject, Identifiable {
+    public var id = UUID()
+    @objc @Published public var leadingIcon: UIImage?
```

Comment:
We'll soon need to add a view as an option to the image (Avatar cell scenario).
Can you create a GitHub issue to track that work?

Reply:
https://github.com/microsoft/fluentui-apple/issues/365